test/test_loadfail.py

```diff
@@ -0,0 +1,58 @@
+#!bin/env python
+
+import subprocess
+import os.path
+import unittest, re
+
+
+class TestSaveLoad(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+
+        subprocess.call('rm -rf remote local 2>> /dev/null', shell=True)
+
+        subprocess.call('mkdir remote; mkdir local', shell=True)
+        subprocess.call('cd remote; mkdir parent; cd parent; git init --bare', shell=True)
+        subprocess.call('cd remote; mkdir child; cd child; git init --bare', shell=True)
+
+        subprocess.call('cd local;  git clone ../remote/parent', shell=True)
+        subprocess.call('cd local;  git clone ../remote/child', shell=True)
+
+        subprocess.call('cd local/parent;  echo "version: 0.1.0" >> .gitproj', shell=True)
+        subprocess.call('cd local/parent;  echo "repos:" >> .gitproj', shell=True)
+        subprocess.call('cd local/parent;  echo "\tc child ../../remote/child" >> .gitproj', shell=True)
+        subprocess.call('cd local/parent;  git add .gitproj; git commit -m "Initial Commit"; git push -u origin master', shell=True)
+
+
+    def test_init(self):
+
+        subprocess.call('cd local/parent;  git project init', shell=True)
+        subprocess.call('cd local/parent;  git add .gitignore; git commit -m ".gitignore"; git push', shell=True)
+
+        output = subprocess.check_output('cd local/parent; ls | grep child | awk \'{print $1}\'', shell=True)
```

Comment:
Maybe worth using double quotes here:
```python
output = subprocess.check_output("cd local/parent; ls | grep child | awk '{print $1}'", shell=True) 
```
Alternatively, although your way is more robust, gnu and bsd grep both support -o:
```python
output = subprocess.check_output('cd local/parent; ls | grep -o child', shell=True) 
```